lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.spec.ts

```diff
@@ -37,18 +57,22 @@ import {
     TagService
 } from '@alfresco/adf-content-services';
 import { MatExpansionPanel } from '@angular/material/expansion';
+import { PredictionStatusUpdate } from '../../../../../../core/src/lib/prediction/interfaces/prediction-status-update.interface';
```

Comment:
This import path doesn't seem correct. I guess it needs to be imported from @alfresco/adf-core

Reply:
fixed, thanks